jsonschema/_utils.py

```diff
@@ -38,6 +38,22 @@ def __repr__(self):
         return repr(self.store)
 
 
+class Cache(object):
```

Comment:
So it looks like `functools32` is py2.7 only (I actually need 2.6 still).

`repos.lru` works in 2.6, but it's not small (https://github.com/repoze/repoze.lru/blob/master/repoze/lru/__init__.py ~373 lines). I notice that currently there are no `install_requires` dependencies.

I could add an install_requires for `version < 3.2` or I could vendor it in `jsonschema/vendor/repos/lru.py`. I usually prefer to add the dependency instead of vendoring it.  Do you have a preference here?